Add release announcements as a separate blog.

cgiradkar opened the pull request:
Old blogs contained some release announcements, hence created another blog at /release. To add a new RA, simple place the markdown file to "release" folder.

closes #117

TomSweeneyRedHat commented:
@gautamkrishnar Thank YOU!
LGTM
@rhatdan PTAL and merge if it LGTY

gautamkrishnar commented:
@TomSweeneyRedHat Pinged you on Redhat Gchat, you wasn't available online 😄. First of all I reallly liked podman, would be happy to join its development. I am now looking for some bugs to squash. 

Anyway i also found some problems in https://github.com/containers/podman.io/ repo. You havent commited the Gemfile and Gemfile.lock is there any specific reason not doing it? Shall i try adding them on a PR?. It will be harder for new contributors to get involved in this project if they dont know jekyll. Also a getting strated section is missing in the readme. I would like to add all these via a PR if you are good with it.

Please let me know if there is any slack/gitter groups available for the community. I would like to join them.

rhatdan commented:
LGTM
@gautamkrishnar It would be great to have your help on Podman.  Right now there is a ton of work to get Podman V2 back to passing tests.  If you grab an e2e test and run it locally and then build a patch to fix the problems in the tool.

Another option is to look for first issues under issues.

gautamkrishnar commented:
@rhatdan Thanks will check that.

cgiradkar committed:
Add release announcements as a separate blog.

Old blogs contained some release announcements, hence created another blog at /release.
To add a new RA, simple place the markdown file to "release" folder.

closes #117

Signed-off-by: Chetan Giradkar <cgiradka@redhat.com>

TomSweeneyRedHat reviewed (commented):
LGTM
